inc/ticket.class.php

```diff
@@ -1187,12 +1187,12 @@ function prepareInputForUpdate($input) {
          if (count($users)) {
             $field = 'users_id';
             foreach ($users as $user) {
-               if (!isset($input['_'.$field.'_'.$t]) || !in_array($user['id'], $input['_'.$field.'_'.$t])) {
+               if (!isset($input['_'.$field.'_'.$t]) || !in_array($user['users_id'], $input['_'.$field.'_'.$t])) {
```

Comment:
Maybe use `$field` instead of raw value?

Reply:
Yes of course I had not seen $field

Reply:
It's easier to see it on the review ;)